### PR TITLE
[#122]FEAT(Class Details Page): add class details page

### PR DESCRIPTION
**Pull request by irvinalfaro11**

### CODE REVIEW

File: src/app/class/class-list/class-list.component.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class ClassListComponent implements OnInit, OnDestroy {
   private dataNameAlias = ['ID', 'Course', 'Term', 'A.Y.', 'Cycle'];
 
   ngOnInit() {
-    this.myClassActionCreator.GetMyClassWithFilter('instructorId','1');
+    this.myClassActionCreator.GetMyClassWithFilter('instructorId','2');
```
Comment (johnhigginsavila) [inline src/app/class/class-list/class-list.component.ts]: Use instructorId on the store

File: src/app/class/class-list/class-page/class-page.component.ts [hunk 2]
```diff
@@ -30,7 +31,8 @@ export class ClassPageComponent implements OnInit {
       .subscribe(params => {
         this.myClassId = params.id;
         this.miscActionCreator.StoreMyClassId(params.id);
-      })
+        (params.id) ? this.studentActionCreator.GetMyClassStudent(params.id) : null;
```
Comment (johnhigginsavila) [inline src/app/class/class-list/class-page/class-page.component.ts:34]: put this inside subscription...

File: src/app/store/action-creators/session.actioncreator.ts [hunk 3]
```diff
@@ -101,6 +101,7 @@ export class SessionActionCreator implements OnDestroy {
     this.authenticationService.SessionDestroy();
     this.ngRedux.dispatch({ type: SESSION_DESTROY_FULFILLED });
     this.ngRedux.dispatch({ type: USER_SESSION_DESTROY });
+    this.router.navigate(['./pages/sign-in']);
```
Comment (johnhigginsavila) [inline src/app/store/action-creators/session.actioncreator.ts]: remove "." in navigate function.

**Review by johnhigginsavila — CHANGES_REQUESTED** [review]

File: src/app/class/class-list/class-list.component.ts (hunk 1 above)
Comment (irvinalfaro11) [reply src/app/class/class-list/class-list.component.ts]: this is cover to the another story

**Review by johnhigginsavila — APPROVED** [review]
- LGTM

**Pull request merged**